config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "bots#index"
-
+  
   resources :bots do
     resources :bot_instances
   end
+  post 'bots/:bot/collaborators', to: 'bots#add_collaborator'
```

Comment:
As a ProTip, you could tack:

```
, as: :add_collaborator
```

to the end of this. That'd get you Rails path helpers, which would clean up things like `bots/_form.html.erb:32` by allowing this:

```
form_tag add_collaborator_path(bot.id) do
```

and other such niceties in other places :)